keyless/pepper/example-client-rust/src/main.rs

```diff
@@ -164,31 +160,31 @@ async fn main() {
         .await
         .unwrap();
     assert_eq!(StatusCode::OK, raw_response.status());
-    let pepper_response = raw_response.json::<PepperResponseV1>().await.unwrap();
+    let pepper_response = raw_response.json::<PepperResponse>().await.unwrap();
     println!();
     println!(
         "pepper_service_response={}",
         serde_json::to_string_pretty(&pepper_response).unwrap()
     );
-    let PepperResponseV1 {
-        signature_encrypted,
-    } = pepper_response;
-    let esk_scalar = esk.derive_scalar();
-    let pepper = ElGamalCurve25519Aes256Gcm::dec(&esk_scalar, &signature_encrypted).unwrap();
-    println!();
-    println!("pepper={:?}", pepper);
-    let claims = jwt::parse(jwt.as_str()).unwrap();
-    println!();
-    println!("Verify the pepper against the server's verification key, part of the JWT, and the actual aud.");
-    let pepper_input = PepperInput {
-        iss: claims.claims.iss.clone(),
-        uid_key: "sub".to_string(),
-        uid_val: claims.claims.sub.clone(),
-        aud: dummy_aud.clone(),
-    };
-    let pepper_input_bytes = bcs::to_bytes(&pepper_input).unwrap();
-    vuf::bls12381_g1_bls::Bls12381G1Bls::verify(&vuf_pk, &pepper_input_bytes, &pepper, &[])
-        .unwrap();
-    println!();
-    println!("Pepper verification succeeded!");
+    // let PepperResponse {
```

Comment:
What's happening here? Is this ready for review?

Reply:
rdy